src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -310,6 +433,73 @@ const FilterableWorkflowTable = ({
                           }
                         },
                       },
+                      {
+                        size: { basis: 400, grow: 0 },
+                        field: 'taskData',
+                        headerRenderer: () => 'Workflow Data',
+                        cellRenderer: ({ rowIndex }) => {
```

Comment:
If you look at the run details page, there's a similar per-task column there with similar contents (Inputs, Outputs, Logs). The ordering of the links, and the spacing between them, is different in the two pages - can you make this one consistent with what's in Run Details?